src/handlers/store-standardised-invoices/get-s3-event-records.ts

```diff
@@ -0,0 +1,26 @@
+import { S3Event, S3EventRecord, SQSRecord } from "aws-lambda";
+
+export const getS3EventRecords = (queueRecord: SQSRecord): S3EventRecord[] => {
```

Comment:
This is a great function and it also used in the extract handler. 
Is there a way it can be popped in the shared/utils/s3 file and imported to be used for both handlers? Happy to pop this in a tech debt ticket if you want to keep this PR just for store-standardised-invoice?